lib/config/header.ex

```diff
@@ -3,17 +3,19 @@ defmodule ScreensConfig.Header do
   Common header type for most screens. Can be configured with a `stop_name` (really an arbitrary
   string, presented as-is) or a `stop_id` (indicating the header should be the name of that stop).
 
+  `read_as` is how the section should be announced in audio readouts. If `nil`, defaults to the
+    configured `title`.
+
   `use ScreensConfig.Header` inserts appropriate `value_from_json` and `value_to_json` clauses to
   encode and decode these variants.
   """
 
   defmodule StopId do
     @moduledoc false
 
-    @type t :: %__MODULE__{stop_id: String.t()}
+    @type t :: %__MODULE__{stop_id: String.t() | nil, read_as: String.t() | nil}
```

Comment:
I think the way this works now with everything technically being optional is a little confusing, and could result in more work for library users (i.e. Screens) to handle the added possibilities of `nil`s. We already have a way to represent what we want for the SL use case, namely `{"stop_name": ""}`.

Reply:
With the empty string for handling empty headers, I've updated this to not have every field as optional. With this setup, I'll make it clear that to have an empty visual header, the screen should be configured with `stop_name: ""`

```
  defmodule StopName do
    @moduledoc false

    @type t :: %__MODULE__{stop_name: String.t(), read_as: String.t() | nil}

    @enforce_keys [:stop_name]
    defstruct @enforce_keys ++ [:read_as]
```

```
  defmodule StopId do
    @moduledoc false

    @type t :: %__MODULE__{stop_id: String.t(), read_as: String.t() | nil}

    @enforce_keys [:stop_id]
    defstruct @enforce_keys ++ [:read_as]
```